### ISSUE TITLE
SuperNotCalledException in RichMediaWebActivity

### ISSUE THREAD

**Issue by litrik** [Pushwoosh/pushwoosh-android-sdk#155]
Versions

- com.pushwoosh:pushwoosh:6.1.1
- com.pushwoosh:pushwoosh-firebase:6.1.1

I have seen the following crash arrive in Crashlytics:

```
Fatal Exception: android.util.SuperNotCalledException: Activity {XXX/com.pushwoosh.inapp.view.RichMediaWebActivity} did not call through to super.onCreate()
       at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3100)
       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3259)
       at android.app.ActivityThread.handleRelaunchActivityInner(ActivityThread.java:5062)
       at android.app.ActivityThread.handleRelaunchActivity(ActivityThread.java:4971)
       at android.app.servertransaction.ActivityRelaunchItem.execute(ActivityRelaunchItem.java:69)
       at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:108)
       at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:68)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1950)
       at android.os.Handler.dispatchMessage(Handler.java:106)
       at android.os.Looper.loop(Looper.java:214)
       at android.app.ActivityThread.main(ActivityThread.java:7073)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:494)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:964)
```

I have seen this on a number of different devices:
- Motorola with Android 9
- Samsung with Android 9
- Huawei with Android 8
 
I haven't been able to reproduce this. It only appeared in our Crashlytics logs. I was hoping the stack trace might be sufficient for you to investigate this further.

**Comment by litrik** [comment]
FYI: This never happened with the previous version (6.1.0)

**Comment by wfhm** [comment]
@litrik Checking it.

**Comment by litrik** [comment]
FYI: Same thing happens with Pushwoosh 6.1.4

**Comment by wfhm** [comment]
@litrik we have been unable to reproduce the issue so far, however we have a theory that it can happen when changing device orientation (from portrait to album or vice versa). Could you please clarify whether your app suggests using album orientation and/or switching between portrait and album? If so, by any chance can you try reproducing the issue with your app by triggering rich medias in different orientations or when changing orientation with an opened rich media?

**Comment by litrik** [comment]
The main activity of the app has android:screenOrientation="userPortrait"

It all sounds a lot like https://stackoverflow.com/questions/40297953/supernotcalledexception-activity-cropimage-did-not-call-through-to-super-onconf

**Comment by wfhm** [comment]
@litrik,

Fixed with the 6.2.3 release.
